nuxt.config.ts

```diff
@@ -311,9 +311,12 @@ export default defineNuxtConfig({
     lazy: true,
     strategy: 'no_prefix',
     detectBrowserLanguage: false,
-    langDir: 'locales',
+    langDir: resolve('locales'),
```

Comment:
Check anither pr to use relative paths herw

Reply:
https://github.com/nimbus-town/nimbus/pull/11

Reply:
Ah this one: https://github.com/nimbus-town/nimbus/pull/11
Thanks, I'll include them.

Reply:
`landDir`  should be `../locales`